compiler/src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -178,6 +178,7 @@ class ScalaSettings extends Settings.SettingGroup with CommonScalaSettings {
   val YtestPickler: Setting[Boolean] = BooleanSetting("-Ytest-pickler", "Self-test for pickling functionality; should be used with -Ystop-after:pickler.")
   val YcheckReentrant: Setting[Boolean] = BooleanSetting("-Ycheck-reentrant", "Check that compiled program does not contain vars that can be accessed from a global root.")
   val YdropComments: Setting[Boolean] = BooleanSetting("-Ydrop-comments", "Drop comments when scanning source files.")
+  val YreadComments: Setting[Boolean] = BooleanSetting("-Yread-comments", "Read comments from tasty.")
```

Comment:
The user-facing concept is `docs` and not `comment`. We only use comments internally for historical reasons (old mistake) and should avoid leaking this name to the users. At some point, we should fix our internal naming scheme.

```suggestion
  val YreadDocs: Setting[Boolean] = BooleanSetting("-Yread-docs", "Read comments from tasty.")
```

Reply:
Now I notice that we should also change `YcookComments` should be renamed to `YcookDocs`.

Reply:
Good catch, now it aligns with existing naming.